src/_path_wrapper.cpp

```diff
@@ -93,10 +93,19 @@ static PyObject *Py_points_in_path(PyObject *self, PyObject *args)
 
 const char *Py_point_on_path__doc__ =
     "point_on_path(x, y, radius, path, trans)\n"
-    "--\n\n";
+    "--\n\n"
+    ".. deprecated:: 3.8\n";
```

Comment:
Doesn't this become `matplotlib._path`, thus private, so we don't need a deprecation for anything in this file?

Reply:
In this case we have `matplotlib.path`, so this is definitely not directly what should be used.

Reply:
OK! Should there be a change note for it?

Reply:
Probably not, I think.